docs/rules/a11y-no-generic-link-text.md

```diff
@@ -1,6 +1,8 @@
-# No Generic Link Text [Deprecated]
+# Disallow generic link text (`github/a11y-no-generic-link-text`)
```

Comment:
Are these title updates auto-generated by the tool? If so what are they based on?

Reply:
Yep, based on the rule name and `meta.docs.description`, customizable here:

https://github.com/bmish/eslint-doc-generator#--rule-doc-title-format